api/src/main/java/org/openmrs/api/db/hibernate/HibernateProviderDAO.java

```diff
@@ -241,9 +241,8 @@ private Junction getNameSearchExpression(String name) {
 	 */
 	@Override
 	public Long getCountOfProviders(String name, boolean includeRetired) {
-		Criteria criteria = prepareProviderCriteria(name, includeRetired);
-		criteria.setProjection(Projections.countDistinct("providerId"));
-		return (Long) criteria.uniqueResult();
+	  Criteria criteria = prepareProviderCriteria(name, includeRetired);
+	  return (long) criteria.list().size();
```

Comment:
Your formatting is different here, are you sure that you ran `mvn clean package` _before_ making the PR?

Reply:
I ran a clean install and package before making a PR :)

Reply:
sorry, I only ran a "mvn install" sorry.

Reply:
I have just realised it :(